lib/autorest/operations/index.d.ts

```diff
@@ -472,6 +475,26 @@ export interface Reports {
     cloneReport(collectionName: string, workspaceId: string, reportKey: string, requestParameters: models.CloneReportRequest, callback: ServiceCallback<models.Report>): void;
 
     /**
+     * @summary Deletes the specified report
```

Comment:
Are you synced? Why does it include DeleteReport API changes? Where those added to Node library?